packages/react-frontend/src/routes/AlbumInfo.jsx

```diff
@@ -145,11 +118,9 @@ export default function AlbumInfo() {
               reviews.map((review) => (
                 <div key={review._id} className="review">
                   <span className="rating-stars">
```

Comment:
Unfortunately because I was unable to communicate well, we have a little merge conflict here. We could probably fix it by either deleting these lines here. or trying to combine it with the one I have merged into develop. Again my bad.